crates/bevy_ecs/src/reflect/bundle.rs

```diff
@@ -85,8 +86,19 @@ impl ReflectBundle {
     }
 
     /// Removes this [`Bundle`] type from the entity. Does nothing if it doesn't exist.
-    pub fn remove(&self, entity: &mut EntityWorldMut) {
-        (self.0.remove)(entity);
+    pub fn remove(&self, entity: &mut EntityWorldMut) -> ReflectBundle {
+        (self.0.remove)(entity)
+    }
```

Comment:
As mentioned on Discord, I think we could move the chaining convenience stuff to these methods rather than on the pointers themselves